yocs_joyop/src/joyop.cpp

```diff
@@ -11,6 +11,8 @@
 #include <std_msgs/String.h>
 #include "boost/thread/mutex.hpp"
 #include "boost/thread/thread.hpp"
+#include <dynamic_reconfigure/server.h>
+#include <gopher_core_msgs/BagJoystickConfig.h>
```

Comment:
This should be called `JoystickConfig`, not `BagJoystickConfig`. The configuration actually has nothing to do with bagging at all. Bagging is just an example user.